std/assembly/arraybuffer.ts

```diff
@@ -31,6 +31,10 @@ export abstract class ArrayBufferView {
     ERROR("missing implementation: subclasses must implement ArrayBufferView#length");
     return unreachable();
   }
+
+  toArrayBuffer(): ArrayBuffer {
```

Comment:
I see why this is useful, but since the same can be achieved by using the normal JS API as of `.buffer.slice`, I *think* it's not strictly necessary and should rather become a helper in a user program.

Reply:
That's true. But manually writing `.buffer.slice`, even though just one line, still feels annoying.

Reply:
Yeah, but that's something that might be better so solve on the application's end I think. Convenient helpers are nice but not exactly the scope of the standard library if there are standard ways already. Having a helper like `viewToBuffer` on the application side, (only) where necessary, seems to be ideal.

Reply:
I see. Will change that.

Reply:
Fixed